app/bundles/UserBundle/Entity/User.php

```diff
@@ -356,6 +363,16 @@ public function getPlainPassword()
         return $this->plainPassword;
     }
 
+    /**
+     * Get automatic password.
+     *
+     * @return string
+     */
+    public function getAutomaticPassword()
+    {
+        return $this->automaticPassword;
+    }
```

Comment:
No need to make a comment that is the same as the method name
```suggestion
    public function getAutomaticPassword(): string
    {
        return $this->automaticPassword;
    }
```

Reply:
Done